### PR TITLE
[botkit] Add Facebook Adapter Bot sample

### PR DESCRIPTION
**Pull request by LCarbonellRod**
### Description

- Add new sample. Echo bot to test the Facebook Adapter.

- A README file is added with the steps to connect the bot with Facebook Api to use the bot.

_**Note**: This PR will fail until the Facebook Adapter nuget package gets published._

### CODE REVIEW

File: samples/csharp_dotnetcore/55.facebook-adapter/README.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+﻿# EchoBot using Facebook Adapter
```
Comment (ceciliaavila) [inline samples/csharp_dotnetcore/55.facebook-adapter/README.md]: Please change the title to match the rest of the samples

Comment (ceciliaavila) [inline samples/csharp_dotnetcore/55.facebook-adapter/README.md]: Please change the title to match with the rest of the adapters.

File: samples/csharp_dotnetcore/55.facebook-adapter/README.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  - Launch Visual Studio
+  - File -> Open -> Project/Solution
+  - Navigate to `BotBuilder-Samples/samples/csharp_dotnetcore/55.facebook-adapter` folder
+  - Select `FacebookAdapterBot.csproj` file
+  - Press <kbd>F5</kbd> to run the project
+
+
+### Connect the bot with Facebook
+
+- Create a Facebook Account for Developers (https://developers.facebook.com/).
```
Comment (ceciliaavila) [inline samples/csharp_dotnetcore/55.facebook-adapter/README.md]: These steps should have numbers, you'll need them to indicate how to set the tokens in appsettings.json

File: samples/csharp_dotnetcore/55.facebook-adapter/wwwroot/default.htm [hunk 3]
```diff
@@ -0,0 +1,420 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Microsoft.Bot.Builder.Facebook.Sample</title>
```
Comment (ceciliaavila) [inline samples/csharp_dotnetcore/55.facebook-adapter/wwwroot/default.htm]: Change the title here.

**Review by ceciliaavila — CHANGES_REQUESTED** [review]

**Review by ceciliaavila — APPROVED** [review]

**Review by matiasroldan6 — APPROVED** [review]

**Pull request closed without merging**